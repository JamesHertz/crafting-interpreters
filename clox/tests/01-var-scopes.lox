var x = 10;
print x;
{
    var y = "He has " + x + " years old";
    print y;
    {
        var x = x + 10;
        print x;
    }
}
