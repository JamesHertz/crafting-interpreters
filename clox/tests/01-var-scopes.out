10
He has 10 years old
20
